ui/src/components/common/Entity.jsx

```diff
@@ -30,10 +30,10 @@ class EntityLabel extends Component {
     if (!entity) {
       return null;
     }
-    const title = entity.getFirst('title') || entity.getCaption();
-    const fileName = entity.getFirst('fileName') || title;
+    const title = entity.getCaption();
+    const fileName = entity.getFirst('fileName') || entity.getFirst('title') || title;
```

Comment:
this can just be `fileName || getCaption()` if we're using a current version of ftm.js.